wiki/Mappers_Guild/tr.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+  - Mapper's Guild
+  - MG
+---
+
+# Mappers' Guild
+
+**Mappers' Guild** osu!'nun [featured artistleri](/wiki/Featured_Artists) için [beatmapler](/wiki/Beatmaps) oluşturmaktan sorumlu bir mapper grubudur. MMO-tarzı bir lonca formatı ile yürütülen lonca, mapperların featured artistlerin beatmaplerinin derecelendirilme prosedürüne katılım sağlamalarına katkıda bulunarak ve mapping ile alakalı görevleri tamamlayarak ödüller kazanmalarını sağlayan bir sistemdir.
+
+Loncanın organizasyonu ve tartışması [osu!dev Discord sunucusundaki](https://discord.gg/ppy) `#mappers-guild` kanalında bulunabilir.
+
+## Sistem
+
+*Mappers' Guild sistemi hakkında güncel bilgiye erişmek için [loncanın web sayfasındaki SSS bölümüne](https://mappersguild.com/faq) bakınız.*
+
+3 ya da daha fazla dereceli mape sahip mapperlar loncaya istedikleri zaman [Mappers' Guild web sayfası](https://mappersguild.com/) üzerinden osu! hesaplarını doğrulatmak suretiyle katılabilirler. Diğer herkes ise web sayfasının görevlerini ve mapleme aktivitesini gözlemlemekte serbesttir.
+
+Mapperlar Mappers' Guild'e bağlı olan featured artistlerin maplerinin derecelendirilme prosedürüne katkıda bulunarak ödüller kazanabilir. Bu, temel olarak beatmap oluşturmaya odaklanır, ancak [modlama](/wiki/Modding) yaparak, [storyboard](/wiki/Storyboards) oluşturarak, ya da lonca içindeki görevleri tamamlayarak katkıda bulunan kullanıcılara da ödül verilir.
+
+### Partiler
+
+Partiler birlikte görev tamamlamak için kayıt yaptıran mapper gruplarıdır. Mapperlar tek seferde istedikleri sayıda partiye katılabilirler.
+
+Özel görevlere katılmak üst lonca rütbesi gerektirir. Bir partinin üst bir lonca rütbesinden sayılması için, üyelerinin en az yarısının gerekli rütbe eşiti ya da üstü olması gereklidir.
+
+### Görevler
+
+Görevler, mapperlar featured artist mapleri oluştururken belirli gereksinimleri sağladıklarında onları ödüllendirir. Görevler genellikle yeni bir featured artist için belirli bir sayıda beatmap oluşturmayı kapsar, ancak bazı görevler ufak numaralar gerektirir.
+
+Her bir görevin parti büyüklüğü/rütbesi şartları, kabul etme bedeli, ve bitiş tarihi vardır. Bir partinin bütün üyeleri görevi kabul etmek için belirtilen bedeli ödemek zorundadır. Bu bedel genellikle tek kişilik görevler ve numaralar gerektiren görevler için daha büyüktür.
+
+Görevlerin süresi 3 aylık inaktiflik sonrasında dolar. Herhangi bir kullanıcı mevcut Mappers' Guild puan bakiyesinden harcayarak görevleri geri açabilir.
+
+Mappers' Guild için görevlere katılım zorunlu değildir. Eğer isterse bir kullanıcı tek başına beatmapler oluşturarak ödüller kazanabilir.
```

Comment:
```suggestion
Mappers' Guild için görevlere katılım zorunlu değildir. Bir kullanıcı, eğer isterse, sadece beatmapler oluşturarak ödüller kazanabilir.
```

Reply:
```suggestion
Mappers' Guild için görevlere katılım zorunlu değildir. Bir kullanıcı, eğer isterse, beatmapleri kendi başına oluşturarak ödüller kazanabilir.
```
your suggestion alters the meaning meant to be given here, this should be better ^